### DISCUSSION TITLE
Is there a way to change the color for `verbatim code`?

### DISCUSSION THREAD

**Discussion by DamonCharlesRoberts** [quarto-dev/quarto-cli#3922] [Q&A] [answered]
I may just be missing it, but I am wondering if I can change the color of the font in the verbatim code for HTML from the purplish color. I am not seeing any obvious CSS options.

**Comment by ghost** [comment]
For normal verbatim text this is achieved by `<code style="color: green;">xyz</code>`. Maybe this helps.

**Comment by cderv** [comment, marked as answer]
If you are referring the highlighting code then this is controlled by highlighting style
https://quarto.org/docs/output-formats/html-code.html#highlighting

You could also create a new style or tweak an existing. Or possibly find the CSS rule that you want to modify and change the rule in a CSS or scss quarto theme
